nipype/scripts/utils.py

```diff
@@ -68,13 +68,35 @@ def add_args_options(arg_parser, interface):
             args["default"] = getattr(inputs, name)
             args["action"] = 'store_true'
 
+        # current support is for simple trait types
+        if not spec.inner_traits:
+            trait_type = type(spec.trait_type.default_value)
+            if trait_type in (str, int, float):
+                args["type"] = trait_type
+        elif len(spec.inner_traits) == 1:
```

Comment:
What if `len(spec.inner_traits) > 1`? is that a possible case?. If so, we should raise an error